supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/statuspolling/JesApiQueryManager.scala

```diff
@@ -53,13 +53,16 @@ class JesApiQueryManager(val qps: Int Refined Positive) extends Actor with Actor
   }
 
   private def handleQueryFailure(failure: JesApiQueryFailed) = failure match {
-    case JesApiQueryFailed(request, _) if request.failedAttempts < maxRetries =>
-      val nextRequest = request.withFailedAttempt
+    case failed: JesApiQueryFailed if failed.query.failedAttempts < maxRetries =>
+      val nextRequest = failed.query.withFailedAttempt
       val delay = nextRequest.backoff.backoffMillis.millis
       context.system.scheduler.scheduleOnce(delay, self, nextRequest)
       ()
-    case JesApiQueryFailed(request, _) =>
-      request.requester ! failure
+    case JesApiQueryStatusFailed(request, cause) =>
+      request.requester ! JesApiQueryStatusFailed(request, cause)
+      ()
+    case JesApiQueryCreationFailed(request, cause) =>
+      request.requester ! JesApiQueryCreationFailed(request, cause)
```

Comment:
I don't think you need to differentiate those 2 cases here as both `JesApiQueryStatusFailed` and `JesApiQueryCreationFailed` extend `JesApiQueryFailed` and you're doing the same thing for both.
It could just be

```scala
case failure: JesApiQueryFailed =>
     request.requester ! failure
```